pkg/gateway/gateway.go

```diff
@@ -52,8 +53,11 @@ func (g *gateway) Run() error {
 			return md
 		}),
 	)
-	dialOption := []grpc.DialOption{grpc.WithInsecure()}
-	protos.RegisterServerHandlerFromEndpoint(ctx, mux, g.opts.Gateway().Endpoints, dialOption)
+	dialOption := []grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())}
```

Comment:
grpc.WithInsecure is deprecated: use WithTransportCredentials and insecure.NewCredentials() instead. Will be supported throughout 1.x.